mpf/core/service_controller.py

```diff
@@ -75,4 +75,5 @@ def get_coil_map(self) -> List[CoilMap]:
 
         # sort by board + driver number
         coil_map.sort(key=lambda x: x[0] + str(x[1].hw_driver.number))
+        coil_map.sort(key=lambda x: len(x[0] + str(x[1].hw_driver.number)))
```

Comment:
Can you add a minimal test for this in test_ServiceMode.py? Just to make sure that it works for a basic example. Everything should be already set up in the test.

Reply:
Hey, I completed the coil test in in  test_ServiceMode.py.